src/errors.h

```diff
@@ -9,8 +9,7 @@
 #define ERR_VIDEO_COLOUR_DEPTH   6
 #define ERR_AUDIO_INIT           7
 #define ERR_AUDIO_RATE           8
-// TODO: Progressively get rid of ERR_OUT_OF_MEMORY as we replace malloc with new
-#define ERR_OUT_OF_MEMORY        9
+//#define ERR_OUT_OF_MEMORY      9   no longer used
```

Comment:
Just remove it completely

Reply:
sure, it was not to "put off" with a missing value :)

Reply:
History of the file will document why the gap is here :)